src/Tizen.Security.DevicePolicyManager/Tizen.Security.DevicePolicyManager/EmailPolicy.cs

```diff
@@ -0,0 +1,121 @@
+﻿/*
+ *  Copyright (c) 2018 Samsung Electronics Co., Ltd All Rights Reserved
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License
+ */
+
+using System;
+
+namespace Tizen.Security.DevicePolicyManager
+{
+    /// <summary>
+    /// The EmailPolicy provides methods to manage email policies.
+    /// </summary>
+    /// <since_tizen> 6 </since_tizen>
+    public class EmailPolicy
+    {
+        private readonly string _popImapPolicyName = "popimap_email";
+        private readonly DevicePolicyManager _dpm;
+        private int _popImapCallbackId;
+
+        private Interop.DevicePolicyManager.PolicyChangedCallback _popImapPolicyChangedCallback;
+        private EventHandler<PolicyChangedEventArgs> _popImapPolicyChanged;
+
+        internal EmailPolicy()
+        {
+        }
+ 
+        internal EmailPolicy(DevicePolicyManager dpm)
+        {
+            _dpm = dpm;
+        }
+
+        /// <summary>
+        /// Checks whether the access to POP or IMAP email is allowed or not.
+        /// </summary>
+        /// <returns>true if the POP or IMAP email is allowed, false otherwise.</returns>
+        /// <since_tizen> 6 </since_tizen>
+        /// <exception cref="ArgumentException">Thrown when failed because of invalid handle of DevicePolicyManager.</exception>
+        /// <exception cref="TimeoutException">Thrown when failed because of timeout.</exception>
+        public bool GetPopImapState()
```

Comment:
I considered using the property all policies. but the GetMessagingState in TelephonyPolicy is needed argument. and I wanted to apply the same format to all policies. so I use method all policies. but Is that better using property policies except the GetMessagingState in TelephonyPolicy?

Reply:
In C# prefer property, if it is a value of object.
Method only use to represent a action on about object

Reply:
We guide to avoid throwing exception from property getter. 

>X AVOID throwing exceptions from property getters. 
    Property getters should be simple operations and should not have any preconditions. If a getter can throw an exception, it should probably be redesigned to be a method. Notice that this rule does not apply to indexers, where we do expect exceptions as a result of validating the arguments.

If you must report exception to app developer, use method. 
but, if you can replace exception with default value like `false` and it is a natural behavior to app developer. we recommand to use property getter.
and this case seem to be better to use property getter